### PR TITLE
tests: addition of test_example_app.py

### LINKED ISSUE #50: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:

- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
* Adds test_example_app.py. (closes #50)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
waiting for #51

**Comment by jirikuncar** [comment]
@hachreak you can rebase now. Thanks for waiting!

**Comment by hachreak** [comment]
ping @jirikuncar

File: tests/test_example_app.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import os
+import subprocess
+
+
+def setup_module(module):
+    """Set up before all tests."""
+    # switch to examples/app.py
+    project_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    exampleappdir = os.path.join(project_dir, 'examples')
+    os.chdir(exampleappdir)
```
Comment (jirikuncar) [inline tests/test_example_app.py]: Shall we store the current dir and chdir back on teardown?

Comment (hachreak) [inline tests/test_example_app.py]: how can I store it?

Comment (jirikuncar) [inline tests/test_example_app.py]: @hachreak I was looking into it and it seems that the cleanest solution would be to convert `setup_module` and `teardown_module` to a yield fixture that we pass to `test_*` function.

Comment (hachreak) [inline tests/test_example_app.py]: yes, I alredy did it in `invenio-records-rest`

**Pull request merged**